packages/auth/src/environment.js

```diff
@@ -19,6 +19,8 @@ module.exports = {
   MINIO_URL: process.env.MINIO_URL,
   INTERNAL_API_KEY: process.env.INTERNAL_API_KEY,
   MULTI_TENANCY: process.env.MULTI_TENANCY,
+  ACCOUNT_PORTAL_URL: process.env.ACCOUNT_PORTAL_URL,
+  SELF_HOSTED: !!parseInt(process.env.SELF_HOSTED),
```

Comment:
In the case of `SELF_HOSTED`, when it's turned off the value is "0" which evaluates to truthy. 
Any alternative you would recommend here? I think the other option is to unset the value if we want to enable cloud, but that seems less deterministic

Reply:
On my view replacement branch I had this issue, as in development we currently set the `SELF_HOSTED` in the .env file, which means that simply unsetting it is a bit of a pain - its easier to override.

I added the following right after the `module.exports` has been inited with values:
```
// convert any strings to numbers if required, like "0" would be true otherwise
for (let [key, value] of Object.entries(module.exports)) {
  if (typeof value === "string" && !isNaN(parseInt(value))) {
    module.exports[key] = parseInt(value)
  }
}
```
as this works permanently across all env variables and will make sure any truthy "0" are converted to real false zeroes.

Reply:
@mike12345567 that's a nice solution! When these branches come together I can simply remove the `parseInt` from above then?

Reply:
I'll merge develop in and remove it then!